docs/Community-Project Committers.md

```diff
@@ -42,3 +42,7 @@
 |  Tianan li     |  Tsinghua University |
 |  Yue Su        |	Tsinghua University |
 |  Hui Da        |	Lenovo              |
+|  Yuan Tian     |  Tsinghua University |
+|  Zesong Sun    |  Tsinghua University |
+|  Kaifeng Xue   |  Tsinghua University |
+|  Tianci Zhu    |                      |
```

Comment:
Do you forget @jack870131?

Reply:
sorry，please give me more information about jack

Reply:
His name is Tsung-Han Tsai